packages/babel-types/src/builders/generated/deprecated.js

```diff
@@ -0,0 +1,251 @@
+/*
+ * This file is auto-generated! Do not modify it directly.
+ * To re-generate run 'make build'
+ */
+
+export { arrayExpression as ArrayExpression } from "./index";
+export { assignmentExpression as AssignmentExpression } from "./index";
+export { binaryExpression as BinaryExpression } from "./index";
```

Comment:
To save some space, this could be
```js
export {
  arrayExpression as ArrayExpression,
  assignmentExpression as AssignmentExpression,
  ...
} from "./index";
```

Reply:
will do